### DISCUSSION TITLE
Video Surveillance in the EFB

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#716] [Ideas]
Would be a very cool feature, provided it doesn't take too much performance. The zibo currently also has it and performance isn't lost.

![image](https://user-images.githubusercontent.com/81443205/155108985-bd215c6f-9c9f-4f55-8e20-828fd95c02a5.png)

Accessible from the main menu, video page
![image](https://user-images.githubusercontent.com/81443205/155109157-7efc117a-d383-4321-adc6-de02ce51188e.png)

(from this video: https://youtu.be/hQKUZHtnUic?t=259)

**Comment by aeroniemi** [comment]
It'd literally just be a photo which seems a bit pointless

**Reply by Simleybob** [reply]
in the zibo, when you open the door it looks open on the screen - so they probably have two photos.

It's very low effort and it could be a nice chuckle, might as well - give it a shot, it won't take that long! lol

**Reply by Simleybob** [reply]
and when you click on the bottom images, they come to the big picture in real life.

**Reply by aeroniemi** [reply]
ngl this sounds like a lot of effort for very little gain

**Reply by Simleybob** [reply]
just say you'll think about it lol, but don't spend time on this if it takes time from the fms.
